tsconfig.json

```diff
@@ -3,7 +3,7 @@
     /* Basic Options */
     "target": "es5",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
     "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
-    "lib": ["es2015", "dom"],                /* Specify library files to be included in the compilation. */
+    "lib": ["es2015", "dom", "es2017.string"],                /* Specify library files to be included in the compilation. */
```

Comment:
Why did this change?

Reply:
removed, it was imported to allow using es7 string features